packages/app/src/specs/CreateSpecModal.vue

```diff
@@ -109,7 +109,7 @@ const helpLink = computed(() => {
 const specFileName = computed(() => {
   const extension = props.gql.currentProject?.fileExtensionToUse ?? 'js'
 
-  const fileName = props.gql.currentProject?.defaultSpecFileName ?? `cypress/e2e/filename.cy.${extension}`
+  const fileName = props.gql.currentProject?.defaultSpecFileName ?? (props.gql.currentProject?.currentTestingType === 'e2e' ? `cypress/e2e/spec.cy.${extension}` : `src/ComponentName.spec.${extension}`)
```

Comment:
Looking through `defaultSpecFileName` in ProjectDataSource, it's a bit strange that we allow `defaultSpecFileName` to be null. Instead, we could just return the default and mark the field as `nonNull`, then we wouldn't have to have defaults on the frontend and backend.

Reply:
@ZachJW34 How would I make `defaultSpecFileName` be a `nonNull` field if `currentProject` could still potentially be null?

Reply:
At this stage, it doesn't make sense for `currentProject` to be null. Even if it was, the frontend code relies on there being a `currentProject` to determine the fileName anyway. The check for `currentProject` is more of a TS safety check, so we could replace the `return null` by throwing an error, or just return the default e2e spec name.

Reply:
@tbiethman now that we are always returning a `defaultSpecFileName` can we remove
```
?? (props.gql.currentProject?.currentTestingType === 'e2e' ? `cypress/e2e/spec.cy.${extension}` : `src/ComponentName.spec.${extension}`)
```